installer-for-dhs/src/main/java/com/marklogic/hub/dhs/installer/command/InstallIntoDhsCommand.java

```diff
@@ -76,7 +77,9 @@ public void run(ApplicationContext context, Options options) {
         );
         commands.add(deployRolesCommand);
 
-        commands.add(new DeployAmpsCommand());
+        DeployAmpsCommand deployAmpsCommand = new DeployHubAmpsCommand();
+        deployAmpsCommand.setExecuteSortOrder(390);
```

Comment:
We always install dhs using  admin user. Not sure if it's needed.

Reply:
I don't think we should rely on the admin roles capability when we know that the real issue is with task prioritization.

Reply:
Let's still just use DeployAmpsCommand here though, not DeployHubAmpsCommand, as the latter is specifically for a limited user.

Reply:
Good catch, updated.